config/default/common/config/metadata/stories/surface_water_extent/step005.md

```diff
@@ -0,0 +1 @@
+The Indus River Valley receives heavy monsoon rains each summer which contribute to the apperance of extensive surface water. These false-color reflectance images from Landsat 8 show water in dark blues. The left image is from 3 May 2023, before the monsoon rains, and the right side is from 7 August 2023. Swiping the bar back and forth reveals the extent of floodwaters along the riverbanks and in the surrounding region.
```

Comment:
"appearance" is misspelled.

It seems to me that the left image (prior to the monsoon in the description) is "wetter" than the right side based on the darker blue (open water) in the images. Am I just confused or can the description be improved?

Reply:
Fixed the typo. I added the wrong link for step 5, so I've updated that now!